src/http/fetch.rs

```diff
@@ -168,6 +210,18 @@ fn get_fetch_options<'js>(
         {
             abort_receiver = Some(signal.borrow().sender.subscribe());
         }
+
+        if let Some(redirect_opt) = get_option::<String>("redirect", &arg_opts, &resource_opts)? {
+            redirect = Some(match redirect_opt.as_str() {
```

Comment:
We can probably do something like:
```rust
let redirect_str = redirect_opt.as_str();
if !matches!(redirect_str, "follow" | "manual" | "error"){
   return Err(....)
}
```

Reply:
We have corrected the problem, please confirm.